### ISSUE TITLE
Centos requires - libicu?

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/postgres-formula#194]
Issue seen on Centos

```
          ID: postgresql-server
    Function: pkg.installed
      Result: False
     Comment: Error occurred installing package(s). Additional info follows:
              
              errors:
                  - Running scope as unit run-21177.scope.
                    Loaded plugins: fastestmirror, versionlock
                    Loading mirror speeds from cached hostfile
                    Resolving Dependencies
                    --> Running transaction check
                    ---> Package postgresql10-server.x86_64 0:10.3-1PGDG.rhel7 will be installed
                    --> Processing Dependency: postgresql10-libs(x86-64) = 10.3-1PGDG.rhel7 for package: postgresql10-server-10.3-1PGDG.rhel7.x86_64
                    --> Processing Dependency: postgresql10(x86-64) = 10.3-1PGDG.rhel7 for package: postgresql10-server-10.3-1PGDG.rhel7.x86_64
                    --> Processing Dependency: libpq.so.5()(64bit) for package: postgresql10-server-10.3-1PGDG.rhel7.x86_64
                    --> Processing Dependency: libicuuc.so.50()(64bit) for package: postgresql10-server-10.3-1PGDG.rhel7.x86_64
                    --> Processing Dependency: libicui18n.so.50()(64bit) for package: postgresql10-server-10.3-1PGDG.rhel7.x86_64
                    --> Running transaction check
                    ---> Package postgresql10.x86_64 0:10.3-1PGDG.rhel7 will be installed
                    --> Processing Dependency: libicu for package: postgresql10-10.3-1PGDG.rhel7.x86_64
                    ---> Package postgresql10-libs.x86_64 0:10.3-1PGDG.rhel7 will be installed
                    ---> Package postgresql10-server.x86_64 0:10.3-1PGDG.rhel7 will be installed
                    --> Processing Dependency: libicuuc.so.50()(64bit) for package: postgresql10-server-10.3-1PGDG.rhel7.x86_64
                    --> Processing Dependency: libicui18n.so.50()(64bit) for package: postgresql10-server-10.3-1PGDG.rhel7.x86_64
                    --> Finished Dependency Resolution
                    Error: Package: postgresql10-server-10.3-1PGDG.rhel7.x86_64 (pgdg10)
                               Requires: libicuuc.so.50()(64bit)
                    Error: Package: postgresql10-server-10.3-1PGDG.rhel7.x86_64 (pgdg10)
                               Requires: libicui18n.so.50()(64bit)
                    Error: Package: postgresql10-10.3-1PGDG.rhel7.x86_64 (pgdg10)
                               Requires: libicu
                     You could try using --skip-broken to work around the problem
                     You could try running: rpm -Va --nofiles --nodigest
```

**Comment by noelmcloughlin** [comment]
Duplicate of #195
